## /bin/sh -e simulate dpatch for prepend_chlog.py
## Build
## chanzap_disable_r2.dpatch by Tzafrir Cohen <tzafrir.cohen@xorcom.com>
## DP: disable the nunfunctional support for R2 in chan_zap
## -- applied upstream for trunk (1.4.x)

Index: asterisk-1.2.24.dfsg+pf.0.1~svn1329/channels/Makefile
===================================================================
--- asterisk-1.2.24.dfsg+pf.0.1~svn1329.orig/channels/Makefile	2007-08-08 14:42:42.000000000 +0200
+++ asterisk-1.2.24.dfsg+pf.0.1~svn1329/channels/Makefile	2007-08-08 14:43:04.000000000 +0200
@@ -107,11 +107,6 @@
   ZAPPRI=-lpri
 endif
 
-ifneq ($(wildcard $(CROSS_COMPILE_TARGET)/usr/lib/libmfcr2.so.1)$(wildcard $(CROSS_COMPILE_TARGET)/usr/local/lib/libmfcr2.so.1),)
-  CFLAGS+=-DZAPATA_R2
-  ZAPR2=-lmfcr2
-endif
-
 ALSA_SRC=chan_alsa.c
 
 ifneq ($(wildcard alsa-monitor.h),)
